pkg/hoist/hoist_launchable.go

```diff
@@ -259,18 +264,18 @@ func (hl *Launchable) Executables(
 			}
 
 			p2ExecArgs := p2exec.P2ExecArgs{
-				Command:          []string{filepath.Join(serviceDir, service.Name())},
+				Command:          []string{filepath.Join(hl.InstallDir(), relativePath)},
 				User:             hl.RunAs,
 				EnvDirs:          []string{hl.PodEnvDir, hl.EnvDir()},
-				ExtraEnv:         map[string]string{launch.EntryPointEnvVar: service.Name()},
+				ExtraEnv:         map[string]string{launch.EntryPointEnvVar: entryPointName},
```

Comment:
confirm it is more desirable to give them `bin__launch` rather than just `bin/launch`? (I have no strong opinion, just want to make sure)